packages/graphback-core/src/graphql/graphqlUtils.ts

```diff
@@ -26,3 +29,24 @@ export const filterObjectExtensions = (types: InputModelTypeContext[]) => types.
 export const createImplementsInterfaceString = (names: string[]) => {
   return `implements ${names.map((name: string) => name).join(' & ')} `;
 }
+
+/**
+ * Collects all GraphQL files in a directory and reads the content into a string.
+ *
+ * @param schemaDir - The directory of the schema file(s)
+ */
+export const buildSchemaText = (schemaDir: string): string => {
```

Comment:
IMHO this should not be in the core. This is very specific for the migration package and it is going to go away.